plugins/WoodTypePreset/presets/wood_type/manifest.json

```diff
@@ -0,0 +1,626 @@
+{
+    "name": "Wood Type",
+    "description": "Adds a new Wood Type. Most things will be forced, while others are optional.\nCurrently has: Planks, Log, Stripped Log, Wood, Stripped Wood, Leaves, Fence, Fence Gate, Slab, Stairs, Trapdoor, and Boat.",
+    "icon": "mdi-palm-tree",
+    "category": "[Advanced]",
+    "requires": {
+        "packTypes": [
+            "behaviorPack",
+            "resourcePack"
+        ],
+        "targetVersion": [
+            ">=",
+            "1.18.30"
+        ],
+        "experimentalGameplay": [
+            "holidayCreatorFeatures"
+        ]
+    },
+    "createFiles": [
+        "presetScript/blockImages.js",
+        [
+            "planks.json",
+            "BP/blocks/{{PRESET_PATH}}/wood_types/{{IDENTIFIER}}/{{IDENTIFIER}}_planks.json",
```

Comment:
Presets should no longer use hardcoded BP/RP folder names; see https://github.com/bridge-core/editor-packages/blob/main/packages/minecraftBedrock/preset/entity/bee/manifest.json#L39